src/app/cmdoptions.cpp

```diff
@@ -372,25 +372,25 @@ QStringList QBtCommandLineParameters::paramList() const
     // torrent paths or URLs.
 
     if (!savePath.isEmpty())
-        result.append(QLatin1String("@savePath=") + savePath.data());
+        result.append(u"@savePath=" + savePath.data());
 
     if (addPaused.has_value())
-        result.append(*addPaused ? QLatin1String {"@addPaused=1"} : QLatin1String {"@addPaused=0"});
+        result.append(u"@addPaused="_qs + (*addPaused ? u"1" : u"0"));
```

Comment:
```suggestion
        result.append(*addPaused ? u"@addPaused=1"_qs : u"@addPaused=0"_qs);
```
It would avoid one more string concatenation in runtime.

Reply:
Ok, done.